Assets/Scripts/SceneController.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class SceneController : MonoBehaviour
+{
+    public class SceneState
+    {
+        public string SceneName { get => _sceneName; }
+    
+        private string _sceneName;
+        public SceneState(string sceneName)
+        {
+            _sceneName = sceneName;
+        }
+    }
+    public static SceneController Instance { get; private set; }
+    public SceneState MainMenu { get => _mainMenu; }
```

Comment:
Bedre om du bruker den enklere syntaksen `public SceneState MainMenu => _mainMenu`.
Enda bedre om du lager en auto-property med en privat setter.

Reply:
gjelder for resten av propertyene og feltene også.